experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -29,7 +28,33 @@ const tag6: RevisionTag = mintRevisionTag();
 const tag7: RevisionTag = mintRevisionTag();
 const tag8: RevisionTag = mintRevisionTag();
 
-const testChanges: [string, (index: number) => SF.Changeset<TestChange>][] = [
+const id0: ChangesetLocalId = brand(0);
+
+/**
+ * @returns Lineage for a mark of `length` targeting cells emptied in `revision`.
+ * Assumes `revision` detaches a contiguous block of cells of length `max + 1` which start with ID 0 and have contiguous IDs.
+ * `id` is the ID of the first cell in the block of the mark with this lineage.
+ */
+function generateLineage(
+	revision: RevisionTag,
+	id: ChangesetLocalId,
+	length: number,
+	max: number,
+): SF.LineageEvent[] {
+	const result: SF.LineageEvent[] = [];
+	if (id > 0) {
+		result.push({ revision, id: brand(0), count: id, offset: id });
```

Comment:
This function generates two lineage entries, whereas the rebase code tries to merge them them when possible. Is this a concern for properly exercising the code being tested?